### PR TITLE
Geneticist is now med

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request

Adjusts the code behind the Geneticist to bring them in line with [Beestation SOP](https://wiki.beestation13.com/view/Department_Standard_Operating_Procedure:_Medical#Geneticist), which means placing them solely under the medical department. 
* Removes the wide array of science access from them, as they don't need this to perform their job
* Removes genetics access from RD, and from skeleton crew list of Roboticist / Scientist
* Changes their paycheck to come exclusively out of the medical budget
* Changes their blurb to only reference the Chief Medical Officer
* Geneticist spawns with med headset instead of the unique med-sci one. 

## Why It's Good For The Game

Code mismatching server policies and staff expectations is bad. 
Giving a job that has such a high potential to attract shitters so much potential access kinda seems like a bad idea.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

image

image

</details>

## Changelog
:cl:
fix: Geneticist flavor blurb now matches current Beestation SOP
tweak: Geneticist is no longer partially paid by Science and no longer has science departmental or radio access
tweak: Science no longer has access to genetics
del: Removes medsci headset from code
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
> Might as well just completely remove the headset from the code tbh.

Yeah, tbh I probably should do that with this. I realized I missed the radio because of your forum comment 😆

**Comment by Poivre-Vert** [comment]
Scientist, roboticist, and RD still have access to genetics on their cards, either during skeleton or normal crew.

**Comment by PowerfulBacon** [comment]
They are medical research

**Comment by Rukofamicom** [comment]
> They are medical research

So is Viro. 
Botany is Plant research
Engineering can generate research points
Mining can rapidly generate discovery points

File: code/modules/jobs/job_types/geneticist.dm [hunk 1]
```diff
@@ -11,15 +11,14 @@
...
 
 	department_flag = MEDSCI
-	departments = DEPT_BITFLAG_MED | DEPT_BITFLAG_SCI
-	bank_account_department = ACCOUNT_MED_BITFLAG | ACCOUNT_SCI_BITFLAG
+	departments = DEPT_BITFLAG_MED
+	bank_account_department = ACCOUNT_MED_BITFLAG
 	payment_per_department = list(
-		ACCOUNT_MED_ID = PAYCHECK_MEDIUM_BY_HALF,  // Paid by med for half
-		ACCOUNT_SCI_ID = PAYCHECK_MEDIUM_BY_HALF   // And paid by sci for half
+		ACCOUNT_MED_ID = PAYCHECK_MEDIUM //Geneticist is under only medbay here
```
Comment (EvilDragonfiend) [inline code/modules/jobs/job_types/geneticist.dm]: redundant comment. just remove it.

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -149,11 +149,7 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	icon_state = "sci_headset"
 	keyslot = new /obj/item/encryptionkey/headset_sci
 
-/obj/item/radio/headset/headset_medsci
-	name = "medical research radio headset"
-	desc = "A headset that is a result of the mating between medical and science."
-	icon_state = "medsci_headset"
-	keyslot = new /obj/item/encryptionkey/headset_medsci
+
```
Comment (EvilDragonfiend) [inline code/game/objects/items/devices/radio/headset.dm]: please remove 2 empty lines.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
code is fine, but not sure the idea.

**Comment by EvilDragonfiend** [comment]
Note that this code doesn't change RD's access for the genetics.

**Comment by PowerfulBacon** [comment]
I mean it's fine if guess since all maps have genetics in medical now anyway, from an SOP stand point this is one of the ones where is it wasn't for that we might as well change the SOP to make sense.
Since they don't actually practically answer to the RD then it's fine, something like brig phys is a role that should answer to both since they are practically responding to both.

**Comment by CydiaLamiales** [comment]
> Note that this code doesn't change RD's access for the genetics.

It did though?
image

**Comment by EvilDragonfiend** [comment]
> > Note that this code doesn't change RD's access for the genetics.
> 
> It did though? image

in their access management.

**Comment by Rukofamicom** [comment]
> Since they don't actually practically answer to the RD then it's fine, something like brig phys is a role that should answer to both since they are practically responding to both.

I can't deny that, and yet they are told via rules and in-game text both that they solely answer to the CMO. There is rarely a case where *any* med staff refuses requests from pretty much anyone regardless of rank on the station though.

> in their access management.

Whoops, I'll get to looking into that.

**Comment by Rukofamicom** [comment]
image

**Review by itsmeow — APPROVED** [review]
Effectively geneticist is a medical role with a ton of extra access, which is kinda meh because they do not need the access for any reason whatsoever.

**Pull request merged**